front/lib/resources/storage/index.ts

```diff
@@ -11,7 +11,7 @@ export const frontSequelize = new Sequelize(
   {
     pool: {
       // Default is 5.
-      max: isDevelopment() ? 5 : 50,
+      max: isDevelopment() ? 5 : 40,
```

Comment:
We could probably reduce even more.

Reply:
It's true that we removed a lot of the promise.all's. Likely safer to do it progressively though